aws-sqs-node/src/utils.ts

```diff
@@ -0,0 +1,61 @@
+import { SQSClient } from '@aws-sdk/client-sqs'
+import 'dotenv/config'
+import pinoLogger, { Logger } from 'pino'
+import { z } from 'zod'
+
+const envSchema = z.object({
+  AWS_ACCESS_KEY_ID: z.string(),
+  AWS_SECRET_ACCESS_KEY: z.string(),
+  AWS_REGION: z.string(),
+  AWS_ACCOUNT_ID: z.string(),
+  QUEUE_NAME: z.string(),
+})
+
+let fetchedEnvs: z.infer<typeof envSchema> | null = null
+
+export const getEnv = () => {
+  if (!fetchedEnvs) {
+    const { AWS_ACCESS_KEY_ID, AWS_SECRET_ACCESS_KEY, AWS_REGION, AWS_ACCOUNT_ID, QUEUE_NAME } = process.env
+    const envs = {
+      AWS_ACCESS_KEY_ID,
+      AWS_SECRET_ACCESS_KEY,
+      AWS_REGION,
+      AWS_ACCOUNT_ID,
+      QUEUE_NAME,
+    }
+    const result = envSchema.safeParse(envs)
```

Comment:
I included this logic so the average user gets an error if they have not configured their AWS env variables, which is needed to make this sample run.